frontend/packages/ceph-storage-plugin/src/types.ts

```diff
@@ -361,7 +361,10 @@ export type StorageClusterKind = K8sResourceCommon & {
     storageDeviceSets: DeviceSet[];
     resources: StorageClusterResource;
     encryption?: {
+      /** @deprecated - enable is deprecated from 4.10 */
```

Comment:
what should be used then ?

Reply:
need this for backward compatibility, i.e ODF 4.9. Will remove this in future